extras/travis_build.sh

```diff
@@ -35,6 +35,8 @@ set -e
 if [[ $OSTYPE = darwin* ]]; then
     export CC=$(ls /usr/local/bin/gcc-* | grep '^/usr/local/bin/gcc-\d$')
     export CXX=$(ls /usr/local/bin/g++-* | grep '^/usr/local/bin/g++-\d$')
+    #Ensure that nosetests path can be found
+    export PATH=$HOME/.local/bin:$PATH
```

Comment:
The ways of installation are curious. The log of the last build shows a successful nose install, but to [yet another destination](https://travis-ci.org/github/nest/nest-simulator/jobs/743943712#L357), so this line should most likely be
```suggestion
    export PATH=/Users/travis/Library/Python/3.9/bin:$PATH
```
```